\documentclass[12pt]{report}
\usepackage[margin=1in]{geometry}

\usepackage{enumerate}

\title{Analyzing the Clinical Microbiome \\ Biological Engineering Thesis Proposal}
\author{Claire Duvallet}
\date{October 11, 2016}

\begin{document}


\maketitle

\begin{abstract}
Analyzing the microbiome is hard. Getting clinical insights from analyses is even harder. I'm gonna do some analyses to give us insight into an under-studied clinical microbial system, do some meta-analyses to get direly-needed biological consensus on gut microbiome and disease, and propose a new tool for analyzing 16S datasets.
\end{abstract}

\tableofcontents

\chapter*{Overall objectives and specific aims}
\subsection*{Overall objectives}
My overall objective is to expand 16S microbiome research along the diseases, batches, and groups of microbes axes.

\subsection*{Specific Aims}
\begin{description}
	\item[Aim 1] Establish a baseline for 16S analyses by deeply characterizing one patient cohort in one disease state.
		\begin{enumerate}[(a)]
			\item  Apply current statistical and biological methods to identify microbial community characteristics associated with gastro-esophogeal reflux and aspiration.
			\item Characterize the under-studied lung and gastric microbiomes.
			\begin{enumerate}
			\item Determine how lung, gastric, and throat microbial communities are related and shared.
			\item Identify clinical modulators of these microbial communities, such as diseases state or medication use.
			\end{enumerate}
		\end{enumerate}
	\item[Aim 2] Expand 16S analyses to multiple diseases and patient cohorts to elucidate microbial characteristics consistently associated with health and disease.
	\begin{enumerate}[(a)]
		\item Collect and process all existing case-control gut microbiome 16S datasets in the literature.
		\item Identify general relationships in gut microbial communities within and across different categories of diseases
		\item Discover consistent microbes associated with health and/or individual diseases
		\item Determine best practices for analyzing case-control clinical 16S microbiome studies
	\end{enumerate}
	\item[Aim 3] Define \textit{microbe sets} to enable generalizable interpretations of case-control microbiome studies.
	\begin{enumerate}[(a)]
	\item Use \textit{a priori} knowledge about biological relationships between microbes to define biologically meaningful \textit{microbe sets}.
	\item Use machine-learning techniques to extract disease-associated groups of microbes from datasets collected in Aim 2.
	\item Use the defined \textit{microbe sets} to perform enrichment analyses on a broad spectrum of case-control microbiome studies.
	\end{enumerate}
\end{description}

\chapter*{Proposal}
\section*{Background and significance}
\textit{Three to five pages}

\subsection*{The microbiome}
The microbiome is a hot hot hot field and we know some stuff but we don't know a lot of stuff too.

\subsection*{Microbiome of the upper gastrointestinal and respiratory tracts}
We don't really know anything bc they weren't included in the HMP and clinicians are bad at big data womp womp.

\subsection*{Aspiration, gastro-esophageal reflux disease, and pulmonary infections}
It's all super complicated, yo. And also super connected wat who knew.

\subsection*{Gut microbiome in disease}
Lots of interest, not that many conclusions.

\subsection*{Existing meta-analyses?}
Haven't found much and haven't been great.

\subsection*{Current practice in case-control microbiome studies}
Eh there isn't really a current practice. Use QIIME and see what it tells you?
Common methods: Alpha diversity, JSD, t-tests, etc
Data processing: different methods (OTU calling, Latin name mapping) lead to drastically different results
Data analysis: lots of multiple corrections to do

\subsection*{Enrichment analysis}
GSEA is commonly used in RNAseq data! Gene databases exist and have been curated into gene sets.
No curated microbe sets. Some existing similar tools: SourceTracker, ImG, ...?

\section*{Research design and methods}
\textit{Six to eight pages}
\subsection*{Aim 1}

\subsubsection*{The dataset}
From clinical collaborator, really cool dataset, etc

\subsubsection{Defining shared microbes across sites}
Come up with a metric based on correlation of abundances

\subsubsection{Finding clinical modulators of these relationships}
Aspiration, reflux severity, PPI usage.

\subsection*{Aim 2}

\subsubsection*{The datasets}
Identified through literature search.
Processed using an in-house pipeline that I've also contributed to.

\subsubsection*{Finding general patterns of disease-microbe relationships}
Process and analyze each dataset individually. Compare results.

Look at: alpha diversity, beta diversity for H-Dis, significant genera.

Also try to see how *disease types* are related - either by comparing their microbial signatures or something...

\subsubsection*{Correcting for batch effects}
Do stuff here

\subsection*{Aim 3}

\subsubsection*{Define microbe sets - a priori from biology}
Using an undergrad and lots of lit searching.

\subsubsection*{Define microbe sets - unsupervised, from results from Aim 2}
And \#machinelearning stuff bam wow wow.

\subsubsection*{Perform enrichment analysis on datasets in Aim 2}
See if results jive with what we found from individual genus-based analyses, if we uncover new "signatures of general types of diseases", etc.

\section*{Preliminary studies}
\textit{Three to four pages}

\subsection*{Aim 1}
\subsubsection*{Microbiome community sharedness}
Look what I can do ma.

\subsubsection*{Modulators of sharedness}
And here!

\subsection*{Aim 2}
\subsubsection*{Datasets}
Check out what I got!
\subsubsection*{Preliminary analyses}
PCA, alpha diversity, comparing microbial signatures, consistency of significant OTUs.

\section*{Conclusion}


\end{document}