\documentclass[12pt]{article}
\usepackage[margin=1in]{geometry}

\usepackage{enumerate}
\usepackage{enumitem}
\setlist{nolistsep}

\title{Analyzing the Clinical Microbiome \\ Biological Engineering Thesis Proposal}

\author{Claire Duvallet}
\date{October 11, 2016}

\begin{document}


\maketitle
\newpage

\begin{abstract}
Analyzing the microbiome is hard. Getting clinical insights from analyses is even harder. I'm gonna do some analyses to give us insight into an under-studied clinical microbial system, do some meta-analyses to get direly-needed biological consensus on gut microbiome and disease, and propose a new tool for analyzing 16S datasets.
\end{abstract}
\newpage

\section{Overall objectives and specific aims}
\subsection{Overall objectives}

Many clinicians and scientists are interested in understanding how the human microbiome relates to health and disease, leading to a recent increase in case-control microbiome studies. However, we still do not have clear consensus about the relationship between the human microbiome and disease. Existing analyses often search for individual disease-associated microbes or genes by studying one patient cohort in one disease state. However, published studies from different patient cohorts often contain contradictory or inconsistent results, likely due to technical and analytical batch effects. In this thesis, I will increase our understanding of the microbiome's clinical relevance by expanding our analyses to span multiple diseases, different patient cohorts, and a variety of microbial markers. I will define baseline approaches to study the clinical microbiome in case-control studies. Then, I will apply these approaches to a comprehensive meta-analysis of existing studies. Finally, I will combine the results from these analyses with \textit{a priori} biological knowledge to define groups of microbes which can be used in enrichment analyses and provid clear biological interpretation. This work will improve our understanding of the clinical relevance of the human microbiome and provide best practices and new tools for analyzing future clinical microbiome studies.

\subsection{Specific Aims}
\begin{description}
	\item[Aim 1] Establish a baseline for 16S analyses by deeply characterizing one patient cohort in one disease state.
		\begin{enumerate}
			\item  Apply standard methods to identify microbial community characteristics associated with gastro-esophogeal reflux disease and aspiration.
			\item Characterize the under-studied lung and gastric microbiomes.
			\begin{enumerate}
			\item Determine how lung, gastric, and throat microbial communities are related.
			\item Identify clinical modulators of lung, gastric, and throat communities.
			\end{enumerate}
		\end{enumerate}
	\item[Aim 2] Expand analyses from Aim 1 to multiple diseases and patient cohorts to identify general microbial characteristics associated with health and disease.
	\begin{enumerate}
		\item Discover microbes consistently associated with health and/or individual diseases.
		\item Identify microbial community signatures associated with physiologically-related diseases.
		\item Determine best practices for analyzing case-control 16S microbiome studies.
	\end{enumerate}
	\item[Aim 3] Define \textit{microbe sets} for generalizable interpretations of case-control microbiome studies.
	\begin{enumerate}
	\item Use \textit{a priori} knowledge about microbes to define biologically-driven \textit{microbe sets}.
	\item Use machine-learning techniques to extract disease-associated \textit{microbe sets} from datasets collected in Aim 2.
	\item Use the defined \textit{microbe sets} to perform enrichment analyses on a broad spectrum of case-control microbiome studies.
	\end{enumerate}
\end{description}
\newpage

\section{Background and significance}
\textit{Three to five pages}

\subsection{The microbiome}
The microbiome is a hot hot hot field and we know some stuff but we don't know a lot of stuff too.

\subsection{The aerodigestive tract}
Clinically, gastric and lung disorders are known to be associated but the nature of the relationship is not fully understood. For example, gastro-esophageal reflux disease (GERD) is associated with many respiratory disorders like asthma and chronic pulmonary disease (REF) and aspirators are known to be at higher risk for respiratory infections. However, the exact mechanisms of these associations remains unclear. Researchers have hypothesized that the lungs and stomach may be physically connected, with gastric contents seeding the lungs and leading to disease, and that their microbiota may be involved in causing or exacerbating disease. However, the microbial communities in human lungs and stomachs are among the least studied, except in a few specific disease (CF REF, H. pylori REF). In fact, until recently medical literature stated that the lungs were sterile and free of bacteria (REF). Neither gastric nor lung sites were included in the Human Microbiome Project, leading to a dearth of studies and data on these important body sites.

\subsubsection{Microbiome of the lung, stomach, and throat/oral sites}

\subsubsection*{Aspiration, gastro-esophageal reflux disease, and pulmonary infections}
It's all super complicated, yo. And also super connected wat who knew.

\subsection*{Gut microbiome in disease}
Lots of interest, not that many conclusions.

\subsection*{Existing meta-analyses?}
Haven't found much and haven't been great.

\subsection*{Current practice in case-control microbiome studies}
Eh there isn't really a current practice. Use QIIME and see what it tells you?
Common methods: Alpha diversity, JSD, t-tests, etc
Data processing: different methods (OTU calling, Latin name mapping) lead to drastically different results
Data analysis: lots of multiple corrections to do

\subsection*{Enrichment analysis}
GSEA is commonly used in RNAseq data! Gene databases exist and have been curated into gene sets.
No curated microbe sets. Some existing similar tools: SourceTracker, ImG, ...?

\section{Research design and methods}
\textit{Six to eight pages}
\subsection{Aim 1}
In Aim 1, I will apply existing analytical methods to deeply characterize the lung, gastric, and throat microbiota in a large patient cohort with aspiration and gastro-esophogeal reflux diseases. I will first determine whether and how much of the lung, gastric, and throat microbial communities are shared across the sites. Then, I will investigate potential clinical modulators of this 'sharedness' - modulators like aspiration disease, reflux severity, and use of proton-pump inhibitors. These analyses will provide a baseline understanding of the poorly studied gastric and lung microbiomes, and may point to potential mechanistic causes or possible interventions for a variety of aerodigestive diseases. Additionally, by deeply characterizating this rich dataset, I will establish best practices for future 16S analyses.

\subsubsection{The dataset}
From clinical collaborator, really cool dataset, etc

\subsubsection{Apply standard approaches to find aspiration and reflux-associated features}
Alpha diversity, p-values, JSD.

\subsubsection{Defining shared microbes across sites}
Prelim results indicated that JSD might have an interesting story, so we dive into it.

First, we will investigate how much of the throat, gastric, and lung microbial communities are shared across sites. From an engineering perspective, the stomach, throat, and lungs can be thought of as different compartments connected by the esophagus and windpipe. (FIGURE) The mass transport between these compartments is regulated by complex physiological mechanisms. Swallowing guides material from the mouth to the stomach, but may dysfunction and allow material to enter the lungs. The esophageal sphincter usually prevents material from leaving the stomach, but in some diseases is dysfunctional. Finally, complex homeostatic mechanisms clear the lungs of foreign bodies and create a very selective environment for microbes in the lungs (REFS REFS REFS). Thus, while the throat, stomach, and lung "compartments" are physically connected, the amount of material, including bacteria, flowing between them is not readily apparent. Before investigating how clinical factors can modulate the microbiota in the stomach, lungs, and throat, we must first determine how much exchange of microbial communities can and does happen between these sites.

Only one previous study has examined the relationship between sites of the upper aerodigestive tract (REF). Bassis et al. found that oral microbial communities were more similar to both stomach and lung communities than nasal communities. However, these authors did not examine the inter-relationships between all sites - their driving hypothesis was that the mouth is the source of the downstream communities. In this work, we will take a broader view: which of the upper aerodigestive body sites exchange microbes, are these the same microbes across multiple patients, and are there phylogenetic patterns for these shared microbes? 

To begin, we must define a metric to quantify how "shared" a microbe is across two sites. A shared microbe must fulfill two characteristics: it must often be present in both sites in patients, and its abundance in the two sites should be correlated. If the abundances are correlated, then this indicates that the sites are directly linked: if there is more of the bacteria in one site, then there will also be more in the other sites because of the exchange between sites. A truly "shared" microbe must also co-occur relatively often - otherwise, it may just be present in both sites by chance. There are thus multiple ways to define the "sharedness" metric: as the percentage of patients who have the microbe in both sites, as the correlation between the abundance of the microbe in one site with its abundance in the other site, or some combination of these two approaches. Because so many microbes in the stomach and lungs are seeded by those in the mouth, simple co-occurence of bacteria is not a sufficient bar to establish "sharedness" (REF and FIG). On the other hand, simply choosing the correlation of abundances in both sites is also not adequate to measure sharedness, since in many people the bacteria may not be present in either or both sites. Thus, we define a cutoff of shared vs. not with the correlation and quantify the degree of sharedness using co-occurence. More specifically, for each microbe, we calculate the non-parametric Spearman rank correlation of the log10(relative abundances) in two sites, using only the patients who have the microbe present in both sites. If this correlation is greater than 0.5, the "sharedness" metric is defined as the percentage of patients with the microbe present in both sites. We calculated sharedness of all microbes in this dataset, and are identifying phylogenetic patterns in the shared microbes of the upper aerodigestive tract (see Results).

\subsubsection{Clinical modulators of aerodigestive community relationships}
Once we identify the baseline exchange between the throat, lungs, and stomach, we can begin to ask which clinical factors module the exchange between sites.

Aspiration, reflux severity, PPI usage.


\subsection*{Aim 2}

\subsubsection*{The datasets}
Identified through literature search.
Processed using an in-house pipeline that I've also contributed to.

\subsubsection*{Finding general patterns of disease-microbe relationships}
Process and analyze each dataset individually. Compare results.

Look at: alpha diversity, beta diversity for H-Dis, significant genera.

Also try to see how *disease types* are related - either by comparing their microbial signatures or something...

\subsubsection*{Identify microbial community signatures associated with diseases}
Do stuff here, with collaborators

\subsubsection*{Determine best practices for case-control}
Main issue: batch effects. 
Do stuff here, with collaborators

\subsection*{Aim 3}

\subsubsection*{Define microbe sets - a priori from biology}
Using an undergrad and lots of lit searching. And ImG and Ilana.

\subsubsection*{Define microbe sets - unsupervised, from results from Aim 2}
And \#machinelearning stuff bam wow wow.

\subsubsection*{Perform enrichment analysis on datasets in Aim 2}
See if results jive with what we found from individual genus-based analyses, if we uncover new "signatures of general types of diseases", etc.

\section*{Preliminary studies}
\textit{Three to four pages}

\subsection*{Aim 1}
\subsubsection*{Microbiome community sharedness}
Look what I can do ma.

\subsubsection*{Modulators of sharedness}
And here!

\subsection*{Aim 2}
\subsubsection*{Datasets}
Check out what I got!
\subsubsection*{Preliminary analyses}
PCA, alpha diversity, comparing microbial signatures, consistency of significant OTUs.

\section*{Conclusion}


\end{document}